docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+
+services:
+
+  mongo:
+    container_name: meraki-db-mongo
+    image: ${MONGO_IMAGE}
+    ports:
+      - "27017:27017"
+    # environment:
+    #   MONGO_INITDB_ROOT_USERNAME: ${MONGO_INITDB_ROOT_USERNAME}
+    #   MONGO_INITDB_ROOT_PASSWORD: ${MONGO_INITDB_ROOT_PASSWORD}
+    volumes:
+      - mongodb:/data/db
+      - mongodb_config:/dataconfigdb
+    networks:
+      - meraki-netrwork
+
+  app:
+    container_name: meraki-app
+    image: workfromcafe/meraki-app
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    ports:
+      - ${PORT}:${PORT}
+    links:
+      - mongo
+    environment: 
+      MONGODB_URL: ${MONGO_DB_URL}
+    restart: always
+    volumes:
+      - .:/app
+      - /app/node_modules
```

Comment:
Hello @sandisunandar99
why is this mounted? is this docker compose for development, then you should change the filename to, docker-compose-dev.yml, What do you think?

Reply:
i aggree with @ayocodingit @sandisunandar99

Reply:
hemm.. i think, this different habit for naming file for local development.
if I make file for development actualy nothing add name dev, because why? 
this comand will often used for when you activated docker development `docker-compose up -d .. ..`
when make name file docker using char `dev` on file, so you mush call comand using like this 
`docker-compose -f docker-compose-dev.yml -d ... ...`
So which one is simpler to use? @ayocodingit

Reply:
> hemm.. i think, this different habit for naming file for local development.
> if I make file for development actualy nothing add name dev, because why? 
> this comand will often used for when you activated docker development `docker-compose up -d .. ..`
> when make name file docker using char `dev` on file, so you mush call comand using like this 
> `docker-compose -f docker-compose-dev.yml -d ... ...`
> So which one is simpler to use? @ayocodingit @firmanjs
> 
> and why this mounted ? this for local development 😆 

For a long naming solution, a makefile is usually created for a shortcut, then just make dev, but it is mounted for dev purposes, so that for production it is enough to just docker-compose without having to docker-compose-prod or docker-compose-production, because docker is the main purpose is production not development. #CMIIW

Reply:
okay ,, next dev